### PR TITLE
Added secret mapping and bootstrap config for staging infrastructure

### PR DESCRIPTION
**Pull request by nitinprabhu**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1216

### Change description ###

- This PR overrides database , blob storage and queue connection string secrets with staging infra ones. 

- Other secrets remain same but needs to be present while overriding.

- This PR is dependent on https://github.com/hmcts/reform-scan-shared-infra/pull/43 to create secrets in key vault.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by nitinprabhu** [comment]
> how will the migrations work? i assume both staging and non staging dbs will have to be updated

good point missed it. @mdayican answered it :)

**Comment by mdayican** [comment]
I guess functional test will fail in aat staging as they are still using aat infra but app will run in staging .
Jenkinsfile_CNP files need to be configured.

**Comment by nitinprabhu** [comment]
> I guess functional test will fail in aat staging as they are still using aat infra but app will run in staging .
> Jenkinsfile_CNP files need to be configured.

good catch @mdayican

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -88,7 +88,7 @@ withPipeline(type, product, component) {
 
   // Vars needed for smoke / functional testing
   env.TEST_STORAGE_CONTAINER_NAME = 'bulkscan'
-  env.TEST_STORAGE_ACCOUNT_URL = 'https://reformscanaat.blob.core.windows.net'
+  env.TEST_STORAGE_ACCOUNT_URL = 'https://reformscanaatstaging.blob.core.windows.net'
```
Comment (mdayican) [inline Jenkinsfile_CNP:91]: may we need 2 different values for pr and nonpr ?

Comment (nitinprabhu) [reply Jenkinsfile_CNP:91]: This is used for non PR functional tests. For PR's we use OSBA

**Pull request merged**